pageserver/src/layered_repository.rs

```diff
@@ -48,9 +48,9 @@ use zenith_utils::lsn::{AtomicLsn, Lsn, RecordLsn};
 use zenith_utils::seqwait::SeqWait;
 
 mod blob;
-mod delta_layer;
-mod filename;
-mod image_layer;
+pub mod delta_layer;
```

Comment:
Those will be used in the uploader to 
* hold the layer names in mem between uploads/downloads
* derive the actual FS path when the upload/download is carried out for a certain layer

https://github.com/zenithdb/zenith/pull/686/files#diff-8ef63341d08bdfccc129283af160e1b6032defcd0bc929974510710cdf3d6478R23-R25
(you might first need to expand `synced_storage.rs` file diff for the link to start to work)